clients/gtest/trmm_gtest.yaml

```diff
@@ -18,7 +18,12 @@ Definitions:
     - { M:  1024, N:  1024, lda:  1024, ldb:  1024 }
     - { M:  2000, N:  2000, lda:  2000, ldb:  2000 }
 
-  - &alpha_range [ 1.0, -5.0 ]
+  - &alpha_range [ 1.0, -3.0 ]
+
+  - &complex_alpha_range
```

Comment:
Daine had previously suggested that we can keep only the complex version and the imaginary portion will be ignored for the real cases. This is a bit cleaner but if you are trying to control the quantity of tests then maybe it should remain split up.

Reply:
I will keep it split to reduce the number of tests